### PR TITLE
[TW-397] Update comments for ForageElement

### PR DESCRIPTION
**Pull request by kimberleehowley**
## What

This PR adds comments to the ForageElement methods. 

## Why

The goal of these comments are to better document the SDK so that it is easier to use. 

## Test Plan

- ❌ |  No unit tests are needed for strictly comment changes. 
- ✅ | The reviewer should manually test the changes in this PR: 

Checkout my branch, and then run: 

```
./gradlew dokkaHtml 
npx http-serve reference-docs
```

### CODE REVIEW

**Review by dleis612 — APPROVED** [review]

File: forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt [hunk 1]
```diff
@@ -30,34 +30,105 @@ data class ForageConfig(
...
+     * @param boxStrokeWidth The scaled pixel size.
+     */
     fun setBoxStrokeWidth(boxStrokeWidth: Int)
+
+    /**
+     * Sets the border thickness of the input field when the field is in focus state.
+     *
+     * @param boxStrokeWidth The scaled pixel size.
+     */
     fun setBoxStrokeWidthFocused(boxStrokeWidth: Int)
```
Comment (djoksimo) [inline forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt:105]: @devinmorgan to verify these descriptions

File: forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt [hunk 2]
```diff
@@ -30,34 +30,105 @@ data class ForageConfig(
...
+
+    /**
+     * Sets an event listener to be fired when the ForageElement is blurred.
+     *
+     * @param l The [SimpleElementListener] to be fired on blur events.
+     */
     fun setOnBlurEventListener(l: SimpleElementListener)
+
+    /**
+     * Sets an event listener to be fired on ForageElement change events.
```
Comment (djoksimo) [inline forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt:129]: Can we specify when change events are fired; 

i.e. when the text inside the input field changes.

**Review by djoksimo — CHANGES_REQUESTED** [review]
Nice stuff! Left a comment about the change listener

File: forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt (hunk 2 above)
Comment (kimberleehowley) [reply forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt]: Sure thing - I just pushed up this change!

**Review by djoksimo — APPROVED** [review]

File: forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt (hunk 1 above)
Comment (kimberleehowley) [reply forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt:105]: @devinmorgan Hello! I'll keep this PR open until Monday. If I don't read any feedback from you before then, I'll assume with Danilo and Danny's ✅  I'm good-to-go. Thank you!

**Pull request merged**